### PR TITLE
update openshift operator with keycloak changes

### PR DESCRIPTION
**Pull request by ommkar-xl**
For the updated image, use
docker pull xldevdocker/release-operator:1.3.0-openshift

### CODE REVIEW

File: release-operator-openshift/digitalai-release/kubernetes/dairelease_cr.yaml [hunk 1]
```diff
@@ -11,15 +11,13 @@ spec:
   HealthProbesReadinessTimeout: 60
   ImagePullPolicy: Always
   ImageRepository: xebialabs/xl-release
-  ImageTag: "10.3.3"
-  # Secrets must be manually created in the namespace
```
Comment (rprasad-digitalai) [inline release-operator-openshift/digitalai-release/kubernetes/dairelease_cr.yaml:15]: Please restore the image pull secret changes.

**Review by rprasad-digitalai — CHANGES_REQUESTED** [review]
Restore the image pull secret

File: release-operator-openshift/digitalai-release/kubernetes/dairelease_cr.yaml (hunk 1 above)
Comment (ommkar-xl) [reply release-operator-openshift/digitalai-release/kubernetes/dairelease_cr.yaml:15]: Not required. The Openshift helm chart values.yaml doesn't have PullSecrets for XL-Release.

Comment (ommkar-xl) [reply release-operator-openshift/digitalai-release/kubernetes/dairelease_cr.yaml:15]: In the case of Openshift, the image pull secret can be used with a service account.

**Review by rprasad-digitalai — APPROVED** [review]
Looks good to me

**Pull request merged**